services/movie_session.py

```diff
@@ -42,3 +42,26 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Consider using `get_object_or_404` before attempting to delete to ensure the session exists and handle the case where it does not.